Movie/Movie/CinemaDescriptionScreen/View/Cells/PosterTableViewCell.swift

```diff
@@ -5,25 +5,24 @@ import UIKit
 
 /// Ячейка с паспортом.
 final class PosterTableViewCell: UITableViewCell {
-
     // MARK: - Private visual components
 
     private var posterImageView = UIImageView()
 
     // MARK: - Public methods
 
-    func configureCell(imageData: Data) {
-        configurePosterImageView(imageData: imageData)
+    func configureCell(imageData: UIImage) {
```

Comment:
```suggestion
    func configureCell(imageData:) {
```
осуждаю получаем в ячейке